.gitignore

```diff
@@ -22,3 +22,7 @@
 config/secrets.yml.key
 /node_modules/
 /public/packs/
+
+# Ignore IDE files
+/.idea
+*.iml
```

Comment:
I see that now! I just set this up on my own machine. Would it not be beneficial to have this on our public repository, though just to stop any future people from having my same issue?

Arguments to disclude change:
 - less bloat (albeit small)
 - forces contribute to configure their machine to best practice
 
 Arguments to include change:
 - less for a potential contributor to worry about
 - slightly lower risk to have these files accidentally committed
 
Personally, I think there is still value in adding the lines, but I can see benefits to not as well. What are your thoughts?

Reply:
As for the omniauth - yes, CRB does have an o-auth setup through Google through the `/admin` route. I'm not entirely sure how to test that it still works without "dry running" it. I'm sure there is a way to test it through minitest or some sort of rails testing suite -- I'm just unfamiliar with how to do that.

Do you have an idea on how to do that?

Reply:
I don't have a strong opinion on that `.gitignore` file. I'm happy knowing that you're aware of the tradeoffs and will go along with either choice.

I don't know how to test CRB's auth via Google. Maybe there's a staging environment? Maybe it can be tested in production as long as there's an easy way to rollback upon failure? Seems like we need an expert to chime in. @belongstorachel or @joncanady, perhaps?

Reply:
There is not a staging environment, it''s all manually tested. I'm not immediately sure how to construct a test outside of something like VCR.